### ISSUE TITLE
I want to report a very important bug!!! It is about the big margin between training and test

### ISSUE THREAD

**Issue by shoutOutYangJie** [YunYang1994/tensorflow-yolov3#206]
I found that if I set batchNorm's trainable as True during the test phass, the result is at normal standard level. I guess checkpoint doesn't save two batchnorm's moving_average parameters. I hope the author can notice this problem. 

I change the code like this:

# 61th line
        pred_sbbox, pred_mbbox, pred_lbbox = self.sess.run(
            [self.pred_sbbox, self.pred_mbbox, self.pred_lbbox],
            feed_dict={
                self.input_data: image_data,
                **self.trainable: True**
            }
        )

**Comment by YunYang1994** [comment]
Thank you, I didn't even notice it !

**Comment by dodogoffy** [comment]
@shoutOutYangJie  what is mean "the result is at normal standard level."?

**Comment by shoutOutYangJie** [comment]
sorry, it means that result is expected.


 
---Original---

**Comment by kfcbla** [comment]
Agree, I am facing the same problem. When I set trainable as False in test phase, the model can't detect anything.

**Comment by shoutOutYangJie** [comment]
> Actually it saved.

yes, I chenk the training code, it has ops_udate. so why does this situation happens?
